docs/MODELS.md

```diff
@@ -16,146 +13,138 @@ The below table will list all supported and unsupported functions of the patcher
+| iMac (21.5-inch, Mid 2010) | `iMac11,2` | ^^ |
+| iMac (27-inch, Mid 2010) | `iMac11,3` | ^^ |
+| iMac (21.5-inch, Mid 2011)<br>iMac (21.5-inch, Late 2011) | `iMac12,1` | ^^ |
+| iMac (27-inch, Mid 2011) | `iMac12,2` | ^^ |
+| iMac (21.5-inch, Late 2012) | `iMac13,1` | [Legacy Metal (macOS 13+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1008) |
+| iMac (27-inch, Late 2012) | `iMac13,2` | ^^ |
+| iMac (21.5-inch, Early 2013) | `iMac13,3` | ^^ |
+| iMac (21.5-inch, Late 2013) | `iMac14,1` | ^^ |
+| iMac (27-inch, Late 2013) | `iMac14,2`<br>`iMac14,3` | ^^ |
+| iMac (21.5-inch, Mid 2014) | `iMac14,4` | ^^ |
+| iMac (Retina 5K, 27-inch, Late 2014)<br>iMac (Retina 5K, 27-inch, Mid 2015) | `iMac15,1` | ^^ |
+| iMac (21.5-inch, Late 2015) | `iMac16,1` | ^^ |
+| iMac (Retina 4K, 21.5-inch, Late 2015) | `iMac16,2` | ^^ |
+| iMac (Retina 5K, 27-inch, Late 2015) | `iMac17,1` | ^^ |
+| iMac (21.5-inch, 2017) | `iMac18,1` | Supported by Apple |
+| iMac (Retina 4K, 21.5-inch, 2017) | `iMac18,2` | ^^ |
+| iMac (Retina 5K, 27-inch, 2017) | `iMac18,3` | ^^ |
+| iMac (Retina 5K, 27-inch, 2019) | `iMac19,1` | ^^ |
+| iMac (Retina 4K, 21.5-inch, 2019) | `iMac19,2` | ^^ |
+| iMac (Retina 5K, 27-inch, 2020) | `iMac20,1`<br>`iMac20,2` | ^^ |
+| iMac Pro (2017) | `iMacPro1,1` | ^^ |
 
 ### Mac Pro
 
-| SMBIOS | Year | Supported | Comment |
-| :--- | :--- | :--- | :--- |
-| MacPro1,1 | Mid-2006 | <span style="color:red"> NO </span> | 32-Bit Firmware limitation |
-| MacPro2,1 | Mid-2007 | ^^ | ^^ |
-| MacPro3,1 | Early 2008 | <span style="color:#30BCD5"> YES </span> | - UHCI/OHCI support in Public Beta, see current issues ([#1021](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021)) <br/>- Potential boot issues with stock Bluetooth card, recommend removing to avoid kernel panics |
-| MacPro4,1 | Early 2009 | ^^ | - Everything is supported as long as GPU is Metal capable <br/> - UHCI/OHCI support in Public Beta, see current issues ([#1021](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021))|
-| MacPro5,1 | Mid-2010, Mid-2012 | ^^ | ^^ |
-| MacPro6,1 | Late 2013 | ^^ | - CPU Power Management currently unsupported<br/> - No DRM support |
+| Model Name | Identifier | Tagged Issues |
+| :--- | :--- | :--- | 
+| Mac Pro (Early 2008) | `MacPro3,1` | Recommend upgrade to Metal GPU<br>[non-Metal GPU (macOS 11+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/108)<br>[USB 1.1 (macOS 13+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021)<br>Remove stock Bluetooth to prevent panics |
+| Mac Pro (Early 2009) | `MacPro4,1` | Recommend upgrade to Metal GPU<br>[non-Metal GPU (macOS 11+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/108)<br>[USB 1.1 (macOS 13+)](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/1021) |
+| Mac Pro (Mid 2010)<br>Mac Pro (Mid 2012) | `MacPro5,1` | ^^ |
+| Mac Pro (Late 2013) | `MacPro6,1` | No CPU power management<br>No DRM support |
```

Comment:
Not a fault of the PR, but can remove the latter (and technically former is fixed kinda, just low power mode is broken)